scripts/call_contracts.py

```diff
@@ -371,7 +369,28 @@ def update_token(
     new_token_data = tuple(value or token_data[i] for i, value in enumerate(params))
     tx_receipt = transact(request_manager.functions.updateToken(token_address, new_token_data))
 
-    print(f"Transaction sent, tx_hash: {tx_receipt['hash']}")
+    print(f"Transaction sent, tx_hash: {tx_receipt['transactionHash'].hex()}")
+
+
+@cli.command("invalidate-fill")
+@click.argument("request-id", type=str, callback=validate_bytes)
```

Comment:
I just noticed that we are using type `str` here but everywhere else type `int` for `request-id`. 
I believe this is due to the change of the `request-id` type in the contract.
Looking at the current contract version the type should be `str` everywhere else. 

This change can be done either in this PR or another one since i don't know the amount of effort required for this change. I leave it to you to decide if you want to include them here or in another PR. 
I am fine with both approaches.

Reply:
> I just noticed that we are using type `str` here but everywhere else type `int` for `request-id`. I believe this is due to the change of the `request-id` type in the contract. Looking at the current contract version the type should be `str` everywhere else.

Exactly. This script has quite some technical debt and it shows. I noticed the same and decided to just do the small change here as was needed for my v1 checks. The other things can be fixed later.